content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
+linkTitle: "Configure outlook email notifications"
+weight: 40
+---
+
+## Configure outlook email notifications for pending pentest findings
+
+In order to configure Outlook to send emails every time a new finding gets created, we need to do the following:
+1. Create a Cobalt connection.
+2. Create an Outlook connection.
+3. Find and select a recipe (2 options):
+  A. [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+  B. [Cobalt > Outlook] When DAST finding is detected, send email notification
+4. Copy and customize the recipe
+  1. Configure which pentest should trigger notifications when findings get created.
+  2. Configure which email address to send the to.
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a Cobalt connection.
```

Comment:
I am having doubts about this step. We already have the connection creation in step 2, and this link here is showing the whole process and recipe.
Looks like a confusing step, and given the duration of the video, it is a waste of the user's configuration time.

Reply:
The thing is that we need both connections, one for Cobalt and the other one for Outlook. Does it make more sense now?

Reply:
oh got it. Thank you.

Reply:
[According to Grayson](https://cobalthq.slack.com/archives/C05EYKJ2JKX/p1727104986073709?thread_ts=1727101885.091429&cid=C05EYKJ2JKX), the Cobalt connection should be set up automatically when they initialize the workspace. I think you can start with creating the Outlook connection. That's what I did in the GitHub guide, anyway.

Reply:
Ok thanks, will remove the Cobalt connection step.